source/databricks/calculation_engine/package/calculation/energy/aggregators.py

```diff
@@ -92,7 +92,7 @@ def _aggregate_per_ga_and_brp_and_es(
     ]
     result = t.aggregate_quantity_and_quality(result, sum_group_by)
 
-    return EnergyResults(result)
+    return EnergyResults(result, True)
```

Comment:
It's hard to guess what a `True` means when reviewing. I suggest this for alle occurrences in the PR:
```suggestion
    return EnergyResults(result, ignore_nullability=True)
```

Reply:
yeah the hint does not show up in github like this 
![image](https://github.com/Energinet-DataHub/opengeh-wholesale/assets/39729843/b70aafbc-67dc-4a35-9af1-05cc59dbf50e)